edx_proctoring/api.py

```diff
@@ -1375,12 +1380,17 @@ def get_attempt_status_summary(user_id, course_id, content_id):
     # practice exams always has an attempt status regardless of
     # eligibility
     if credit_service and not exam['is_practice_exam']:
-        credit_state = credit_service.get_credit_state(user_id, unicode(course_id))
+        credit_state = credit_service.get_credit_state(user_id, unicode(course_id), return_course_info=True)
         if not _check_eligibility_of_enrollment_mode(credit_state):
```

Comment:
Can we add the `has_due_date_passed` check on the course_end_date here and return None if it has passed?